File: JavaScript/Beginner_Series_2_Clock-8_kyu.js
// https://www.codewars.com/kata/55f9bca8ecaa9eac7100004a
// 2023-03-09T11:06:06.719+0000
function past(h, m, s){
  let seconds = s + 60 * (m + 60 * h);
  return seconds * 1000;  //1s = 1000ms
}

File: Prolog/Beginner_Series_2_Clock-8_kyu.pl
% https://www.codewars.com/kata/55f9bca8ecaa9eac7100004a
% 2023-05-22T11:42:00.397+0000
past(H, M, S, Result) :- 
  Result is H * 3600000 + M * 60000 + S * 1000.